contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -54,6 +54,12 @@ NXmpes(NXobject):
         
         .. _ISO 18115-1:2023: https://www.iso.org/standard/74811.html
         .. _Clause 11: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:sec:11
+    (NXcoordinate_system):
```

Comment:
NXcoordinate_system_set to hold various NXcoordinate_system instaces

Reply:
Generally I agree, but at the first workshop there was the idea discussed to have a common place to find different coordinate systems and transformations. I thought such a collection could then be such a place.

Reply:
NXinstrument is also just a container without anything linking different elements, no?

Reply:
Let's just go with coordinate system set then.

Reply:
Shall we always give it the same nam? Like geometries(NXcoordinate_system_set).